jdaviz/app.py

```diff
@@ -516,53 +515,51 @@ def _link_new_data(self, reference_data=None, data_to_be_linked=None):
             self.data_collection.add_link(LinkSameWithUnits(wc_old, wc_new))
             return
 
-        try:
-            if linked_data.meta.get("Plugin", None) == 'GaussianSmooth':
-                raise AttributeError
-            dc.add_link(WCSLink(ref_data, linked_data))
-        except (AttributeError, IncompatibleWCS):
```

Comment:
this PR just removes this block entirely (the rest of the diff is changing the indentation and removing no-longer-needed imports).  Imviz skips this method entirely with an if-statement at the top, and currently no other config should require spatial WCS-linking (although we should confirm there are no cases in cubeviz - with a single cube restriction - or mosviz that are affected).

The WCS is still loaded and should be available to mouseover coordinates, etc, but by linking by pixel instead of WCS we avoid bottlenecks which can introduce significant lag.